apps/test-viewer/src/components/Viewer.tsx

```diff
@@ -42,6 +42,7 @@ export function Viewer() {
       }}
       mapLayerOptions={{ BingMaps: { key: "key", value: ApiKeys.BingMapsKey } }}
       tileAdmin={{ cesiumIonKey: ApiKeys.CesiumKey }}
+      theme="os"
       backendConfiguration={{
```

Comment:
should be able to delete the backConfig prop, itll auto point to gpb v4 with the official release

Reply:
Removed